pkg/work/spoke/spokeagent.go

```diff
@@ -87,7 +87,9 @@ func (o *WorkAgentConfig) RunWorkloadAgent(ctx context.Context, controllerContex
 	if err != nil {
 		return err
 	}
-	spokeWorkInformerFactory := workinformers.NewSharedInformerFactory(spokeWorkClient, 5*time.Minute)
+	// resync with a small interval could result in performance issue when the number of appliedmanifestworks
+	// is large.
+	spokeWorkInformerFactory := workinformers.NewSharedInformerFactory(spokeWorkClient, 21*time.Hour)
```

Comment:
24 hours?

Reply:
If both manifestwork informer and appliedmanifestwork informer resync every 24 hours, they may resync at the same time that will result in a huge peak. However, if they have different resync intervals, it will bring less pressure to the work agent.

Reply:
could we add some commenst on this?

Reply:
Comment added.